doc/fitting.rst

```diff
@@ -15,59 +15,17 @@ when using complicated constraints or comparing results from related fits.
 
 
 The :func:`minimize` function
-===============================
+=============================
+
+.. currentmodule:: lmfit.minimizer
```

Comment:
If we do make the change, then all the module references in the other documentation files should be changed to match this.